### PR TITLE
add title to checkbox

### PR DESCRIPTION
**Pull request by jhsu**
use `title` on the wrapper element so that a tooltip can be shown on hover.

### CODE REVIEW

File: src/components/Checkbox/Checkbox.jsx [hunk 1]
```diff
@@ -85,6 +89,8 @@ const Checkbox = createClass({
 			...passThroughs
 		} = this.props;
 
+		const { title } = passThroughs;
```
Comment (sodiumjoe) [inline src/components/Checkbox/Checkbox.jsx]: why not just add to the destructure statement above?

Comment (jhsu) [reply src/components/Checkbox/Checkbox.jsx]: because we still want to pass it to the input

Comment (sodiumjoe) [reply src/components/Checkbox/Checkbox.jsx]: i think it still gets omitted by `omitProps`, since it's declared in proptypes.

Comment (jhsu) [reply src/components/Checkbox/Checkbox.jsx]: you're right, let me look into it

**Pull request merged**